reframe/core/logging.py

```diff
@@ -187,9 +304,21 @@ def __init__(self, logger=None, check=None):
         super().__init__(
             logger,
             {
-                'check_name': check.name if check else 'reframe',
+                'check_name': 'reframe',
                 'check_jobid': '-1',
-                'check_info': check.info() if check else 'reframe',
+                'check_info': 'reframe',
+                'check_system': None,
+                'check_partition': None,
+                'check_environ': None,
+                'check_outputdir': None,
+                'check_stagedir': None,
+                'check_perf_logfile': None,
+                'check_perf_var': None,
+                'check_perf_value': None,
+                'check_perf_ref': None,
+                'check_perf_lower_thres': None,
+                'check_perf_upper_thres': None,
+                'data-version': reframe.VERSION,
```

Comment:
I am not using `data-version`, but perhaps I could. @victorusu What is the intention for that?